site/content/contribute/server/developer-setup/ubuntu.md

```diff
@@ -24,6 +24,7 @@
     ```sh
     sudo apt-get install -y build-essential
     wget https://storage.googleapis.com/golang/go1.12.linux-amd64.tar.gz
+    sudo rm -rf /usr/local/go
```

Comment:
Can you replicate this to `arch.md` and `centos.md`?

Reply:
Sure.

Reply:
The command only makes sense on CentOS, Arch uses pacman which takes care of the check.